packages/imperative/CHANGELOG.md

```diff
@@ -2,9 +2,15 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Resolved bug that resulted in each plugin to have identical public registries regardless of actual installation location/reference
+- BugFix: Resolved bug that resulted in every plugin to have the same registry as the first if multiple were installed in the same command
```

Comment:
- We should clarify what we mean by "the first" and "multiple": " .. to have the same registry as the first plug-in (?) if multiple plug-ins (?) were ... "
- Plug-in should be hyphenated, per the Zowe doc style
- Sentence should end w/ a period
- Also add an issue or PR link to the changelog item